client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -84,12 +119,11 @@ const DomainPicker: FunctionComponent< Props > = ( {
 			<PanelBody>
 				<PanelRow className="domain-picker__panel-row">
 					<div className="domain-picker__header">
-						<div className="domain-picker__header-title">{ NO__( 'Choose a new domain' ) }</div>
-						<Button className="domain-picker__close-button" isTertiary onClick={ () => onClose() }>
-							{ NO__( 'Skip for now' ) }
-						</Button>
+						<div className="domain-picker__header-title">{ NO__( 'Choose a domain' ) }</div>
+						<CloseButton onClose={ () => onClose() } />
```

Comment:
More a question than a review comment: is there any reason nowadays to worry about anonymous functions in props being an anti-pattern, or to prefer the following?

`<CloseButton onClose={ onClose } />`

Just asking for my own knowledge :)

Reply:
I think the latter should be fine since we don't need to pass anything into `onClose()` (that's the only reason I can think of in functional components for using an arrow function here).